README

```diff
@@ -195,3 +195,24 @@ Further Information
 -------------------
 http://igs.bkg.bund.de/index_ntrip.htm
 euref-ip@bkg.bund.de
+
+
+Matternet Addendum
+------------------
+We have now forked this open source software in our github.
+
+Expected usage:
+
+./ntripclient -s <server> -m <mount point> -r <port> -u <Matternet username>
```

Comment:
Is the NMEA GGA sentence required? Will passing -n followed by null function as if no GGA was specified?

Reply:
Is NMEA GGA sentence required? No.
Passing -n with no argument would be the same as not having the -n option.